MedallionShell.Tests/GeneralTest.cs

```diff
@@ -582,10 +582,8 @@ public void TestProcessKeepsWritingAfterOutputIsClosed()
                 command.StandardInput.WriteLine(new string('a', i));
             }
 
-            // workaround for https://github.com/mono/mono/issues/18279; so far
-            // I've encountered this only on Mono Linux
-            if (PlatformCompatibilityHelper.IsMono
-                && !RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            // TODO: This used to be a workaround for https://github.com/mono/mono/issues/18279
```

Comment:
Let me know if you want to keep this comment. FWIW I'll invert this `if` regardless after the first round of review.